felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/rapidsrivers/redis/RedisStore.kt

```diff
@@ -43,25 +51,65 @@ class RedisStore(
                         }
                         null
                     }
-            }.mapKeys {
-                it.key.removePrefix("${keyPrefix.name}$keyPartSeparator")
-            }.also {
-                sikkerLogger.debug("Getting all from redis: $it")
             }
     }
 
-    fun set(
-        key: RedisKey,
+    fun lesResultat(transaksjonId: UUID): JsonElement? = resultatKey(transaksjonId).les()
+
+    fun lesFeil(transaksjonId: UUID): JsonElement? = feilKey(transaksjonId).les()
+
+    fun skrivMellomlagring(
+        transaksjonId: UUID,
+        key: Key,
         value: JsonElement,
     ) {
-        sikkerLogger.debug("Setting in redis: ${key.toStoreKey()} -> ${value.toPretty()}")
+        mellomlagringKey(transaksjonId, key).skriv(value)
+    }
 
-        redis.set(key.toStoreKey(), value.toString())
+    fun skrivResultat(
+        transaksjonId: UUID,
+        value: JsonElement,
+    ) {
+        resultatKey(transaksjonId).skriv(value)
+    }
+
+    // TODO Skriv feil for hver key separat
```

Comment:
Tar denne i en egen PR for å unngå å touche mange filer og endre noe logikk samtidig.